src/components/CardTitle.js

```diff
@@ -1,61 +1,76 @@
-                <div>
-                  <img className="rounded-lg" src="https://s.w-x.co/in-heat_14.jpg?crop=16:9&width=640&format=pjpg&auto=webp&quality=60" />
-                </div>
-                <div className="">
-                  <p>
-                    Heatwaves Return! To Grip Parts of Maharashtra, Madhya
-                    3-5
-                  </p>
-                </div>
-              </div>
-              <div className="">
-                <div>
-                  <img className="rounded-lg" src="https://s.w-x.co/in-heat_14.jpg?crop=16:9&width=640&format=pjpg&auto=webp&quality=60" />
-                </div>
-                <div className="">
-                  <p>
-                    Heatwaves Return! To Grip Parts of Maharashtra, Madhya
-                    Pradesh, Uttar Pradesh, Punjab, Rajasthan, Haryana from June
-                    3-5
-                  </p>
-                </div>
-              </div>
-              <div className="">
+  const [showMoreButton, setShowMoreButton] = useState(data.data.length > 4);
+
+  const loadMoreData = (
+    <>
+      <div className="grid lg:grid-cols-4 gap-12 mt-8">
+        {data.data.map((item, i) => {
+          return (
+            i > 4 && (
+              <>
                 <div>
-                  <img className="rounded-lg" src="https://s.w-x.co/in-heat_14.jpg?crop=16:9&width=640&format=pjpg&auto=webp&quality=60" />
+                  <div>
+                    <img className="rounded-lg md:h-[15vh]" src={item.image} />
+                  </div>
+                  <div className="mt-3">
+                    <p className="md:text-[14px] text-[10px] hover:text-blue-500 cursor-pointer">
+                      {item.title}
+                    </p>
+                  </div>
                 </div>
-                <div className="">
-                  <p>
-                    Heatwaves Return! To Grip Parts of Maharashtra, Madhya
-                    Pradesh, Uttar Pradesh, Punjab, Rajasthan, Haryana from June
-                    3-5
-                  </p>
+              </>
+            )
+          );
+        })}
+      </div>
+      {!showMoreButton && (
+        <button
+          className=" inline-block mt-6 rounded-full px-6 py-2.5 bg-[#1B4DE4] text-white font-medium text-xs leading-tight  hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out"
+          onClick={() => {
+            setShowMoreButton(true);
+          }}
+        >
+          Show Less
+        </button>
+      )}
+    </>
+  );
+
+  return (
+    <div className="bg-white lg:w-[52%] lg:ml-[12.5%] w-9xl ml-[3.3%] rounded-b-lg">
+      <div className="p-4 ">
+        <div className="md:grid grid-cols-4 md:gap-12 flex flex-row gap-2 overflow-auto overflow-y-hidden scrollbar-hide scroll-smooth">
+          {data.data.map((item, i) => {
+            return (
+              i < 4 && (
```

Comment:
Map is not the correct function to be used here. Use a while or something and assign it into an variable and use that here.